src/transformers/models/patchtst/__init__.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import TYPE_CHECKING
+
+# rely on isort to merge the imports
+from ...utils import OptionalDependencyNotAvailable, _LazyModule, is_torch_available
+
+
+_import_structure = {
+    "configuration_patchtst": [
+        "PATCHTST_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "PatchTSTConfig",
+    ],
+}
+
+try:
+    if not is_torch_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_patchtst"] = [
+        "PATCHTST_PRETRAINED_MODEL_ARCHIVE_LIST",
+        "PatchTSTModel",
+        "PatchTSTPreTrainedModel",
+        "PatchTSTForPrediction",
+        "PatchTSTForForecasting",
+        "PatchTSTForMaskPretraining",
+        "PatchTSTForRegression",
+        "PatchTSTForClassification",
+    ]
+
+
+if TYPE_CHECKING:
+    from .configuration_patchtst import PATCHTST_PRETRAINED_CONFIG_ARCHIVE_MAP, PatchTSTConfig
+
+    try:
+        if not is_torch_available():
+            raise OptionalDependencyNotAvailable()
+    except OptionalDependencyNotAvailable:
+        pass
+    else:
+        from .modeling_patchtst import (
+            PATCHTST_PRETRAINED_MODEL_ARCHIVE_LIST,
+            PatchTSTForClassification,
+            PatchTSTForForecasting,
+            PatchTSTForMaskPretraining,
```

Comment:
Maybe for consistency with `BertForPretraining`, we can call this one just `PatchTSTForPretraining`?

Reply:
Also, fun fact, Bert uses `BertForPreTraining`, rather than `BertForPretraining` :p